jormungandr/src/main.rs

```diff
@@ -544,10 +537,8 @@ fn initialize_node() -> Result<InitializedNode, start_up::Error> {
     let settings = raw_settings.try_into_settings(&init_logger)?;
     let mut services = Services::new(logger.clone());
 
-    services.spawn_try_future("sigint_watcher", move |_info| {
-        signal::ctrl_c().map_err(|_| ())
-    });
```

Comment:
We're losing Ctrl+Break handling on Windows here. I think we should handle it too, but move this watcher into the non-Unix implementation of `init_os_signal_watchers`.

Reply:
By the way, it is starting to look like a good time to migrate to GitHub Actions entirely and do testing on Windows too.